src/main/java/appeng/hooks/WrenchHook.java

```diff
@@ -57,6 +60,10 @@ public static InteractionResult onPlayerUseBlock(Player player,
                     return InteractionResult.FAIL;
                 }
 
+                // Play a popping sound
```

Comment:
I think playing the 'Pop' sound that item frames make when you rclick would be appropriate here. Dissasembling with a wrench should make a sound for any given item, and this PR only specifies playing a sound for facades & not terminals, etc and therefore would only play a sound when dissasembling facades, not terminals etc

I'm open to suggestions on which sound should be played when dissasembling